src/builder.rs

```diff
@@ -539,7 +543,13 @@ impl<'a, 'gcc, 'tcx> BuilderMethods<'a, 'tcx> for Builder<'a, 'gcc, 'tcx> {
     }
 
     fn fmul(&mut self, a: RValue<'gcc>, b: RValue<'gcc>) -> RValue<'gcc> {
-        a * b
+        let i=a * b;
```

Comment:
We might as well switch to `Context::new_binary_op()` to set the location directly.